dkist/io/asdf/converters/dataset.py

```diff
@@ -63,6 +63,8 @@ def to_yaml_tree(self, dataset, tag, ctx):
                              "it was not constructed from a set of FITS files.")
         node = {}
         node["meta"] = dataset.meta or {}
+        # If the history key has been injected into the meta, do not save it
+        node["meta"].pop("history", None)
```

Comment:
Won't this mean the history is lost once someone writes a new asdf file? Or is that the point?

Reply:
Yeah. Especially given we don't have a write method at the moment.

Reply:
Fair point.